### DISCUSSION TITLE
Response body representation

### DISCUSSION THREAD

**Discussion by AlmogCabessa** [projectdiscovery/httpx#1508] [Q&A]
Is it possible to save the response body preview with the tags and everything? for example if I'm reaching to github.com I want to see the body like in the picture instead of receiving Skip to content Toggle navigation Sign in Product Actions Automate any workflow Packages Host and m"" 

[screenshot.pdf](https://github.com/projectdiscovery/httpx/files/13800965/screenshot.pdf)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
